charts/prompt-keda/templates/service-init.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: init-{{ include "prompt-keda.fullname" . }}
+  annotations:
+    argocd.argoproj.io/hook: Sync
```

Comment:
Make sense and will work.  Did you consider `PreSync` here and `Sync` for the Scaled Job?

Reply:
My understanding from the docs is that `PreSync` runs _before_ any config changes are applied. If that's correct, then this does need to be `Sync`.

Reply:
Ok I understand now b/c of the config and successful sync aspect.  Thanks!